src/Main.java

```diff
@@ -0,0 +1,7 @@
+package src;
+
+public class Main {
+    public static void main(String[] args) {
+
```

Comment:
Hier mag je pokemons instantieren en methodes van die pokemons aanroepen. 

Kijk ook wat het verschil is als je een `Pokemon charizard = new FirePokemon(...)` of een `FirePokemon charmeleon = new FirePokemon(...)` maakt en daar vervolgens de methodes van aan gaat roepen.